src/vs/platform/terminal/common/terminal.ts

```diff
@@ -3,9 +3,333 @@
  *  Licensed under the MIT License. See License.txt in the project root for license information.
  *--------------------------------------------------------------------------------------------*/
 
+
 import { Event } from 'vs/base/common/event';
-import { IProcessEnvironment } from 'vs/base/common/platform';
+import { IDisposable } from 'vs/base/common/lifecycle';
 import { URI } from 'vs/base/common/uri';
+import { IProcessEnvironment, OperatingSystem } from 'vs/base/common/platform';
+import { IEnvironmentVariableInfo } from 'vs/platform/terminal/common/environmentVariable';
+
+export interface IProcessDataEvent {
+	data: string;
+	sync: boolean;
+}
+
+export type FontWeight = 'normal' | 'bold' | number;
```

Comment:
Still seems like too much is here, `FontWeight`, `ITerminalConfigHelper`, `ITerminalFont` definitely shouldn't be here, maybe `ITerminalConfiguration` too, etc.